docs/scripts_md/imaging_upload_file.md

```diff
@@ -0,0 +1,93 @@
+# NAME
+
+imaging\_upload\_file.pl -- a single step script for the imaging pre-processing
+and insertion pipeline sequence
+
+# SYNOPSIS
+
+perl imaging\_upload\_file.pl &lt;/path/to/UploadedFile> \`\[options\]\`
+
+Available options are:
+
+\-profile      : name of the config file in
+                `../dicom-archive/.loris_mri`
+
+\-upload\_id    : The upload ID of the given scan uploaded
+
+\-verbose      : if set, be verbose
+
+# DESCRIPTION
+
+The program does the following
+
+\- Gets the location of the uploaded file (.zip,.tar.gz or .tgz)
+
+\- Unzips the uploaded file
+
+\- Sources the Environment
```

Comment:
I don't think the script sources the environment file (haven't seen anything in the script that suggests it). Should probably remove this line.